app/tests/test_assets_endpoint.py

```diff
@@ -659,15 +660,20 @@ def setUp(self):  # pylint: disable=invalid-name
         client_login(self.client)
         self.maxDiff = None  # pylint: disable=invalid-name
 
+    @mock_s3_asset_file
     def test_asset_endpoint_delete_asset(self):
         collection_name = self.collection.name
         item_name = self.item.name
         asset_name = self.asset.name
         path = f'/{STAC_BASE_V}/collections/{collection_name}/items/{item_name}/assets/{asset_name}'
+        # Check that the file exists on s3
+        self.assertS3ObjectNotExists(path)
```

Comment:
Here you probably want `assertS3ObjectExists` and not `assertS3ObjectNotExists` ;-)